.azure-pipelines/Windows-CI.yml

```diff
@@ -41,7 +41,7 @@ jobs:
 
   - script: |
       conda create --yes --quiet --name py$(python.version) python=$(python.version)
-      conda install -n py$(python.version) -y -c conda-forge numpy libprotobuf=3.16.0
+      conda install -n py$(python.version) -y -c conda-forge numpy libprotobuf
```

Comment:
Conda-forge's Protobuf hasn't supported 4.21 yet. That's why Windows-CI failed in this PR.